testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -16,6 +16,14 @@
                 "Method added to interface"
             ]
         },
+        {
+            "type": "org.gradle.api.tasks.diagnostics.ProjectReportTask",
+            "member": "Class org.gradle.api.tasks.diagnostics.ProjectReportTask",
+            "acceptation": "Adding method to TaskInternal",
+            "changes": [
+                "Default method has been added in implemented interface"
+            ]
+        },
```

Comment:
❓ This accepted API change on `ProjectReportTask` seems suspicious, what about it?

Reply:
I'm not sure why, but this seems to be necessary.  Without it, even after a refreeze, the binary compatibility report complains that `Class org.gradle.api.tasks.diagnostics.ProjectReportTask: Is not binary compatible.`